presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -232,6 +233,15 @@ public void testColumnComment()
         assertUpdate("DROP TABLE test_column_comment");
     }
 
+    @Override
+    protected TestTable createTableWithDefaultColumns()
+    {
+        return new TestTable(
+                mysqlServer::execute,
+                "tpch.table",
+                "(a bigint, b bigint default 40)");
```

Comment:
this should be passed as parameter. Maybe instead of having `createTableWithDefaultColumns` maybe `Abstract*SmokeTest` should have a method `getSqlExecutor` that you could simply use in `testInsertForDefaultColumn` implementation (it looks like all `createTableWithDefaultColumns` are using same table DDL).

Reply:
But not all connectors which has support for inserting default values would be requiring `getSqlExecutor`. Plus we might not need this abstract method `createTableWithDefaultColumns` once we have enabled adding default value as a column property (like we did for adding `NOT NULL` constraint)

Reply:
> once we have enabled adding default value as a column property (like we did for adding NOT NULL constraint)

@Praveen2112 i don't see this happening in any foreseeable future, really.
Imagine this requires Presto expression translation to other database dialect, with function mapping, etc.

We may _want_ to add this for Hive (because Hive supports DEFAULT and we cannot leverage Hive runtime for computing it, as we don't use the runtime).
But this is a very long way even for single case like Hive, not to mention MySQL, PoostgreSQL and all other JDBC connectors.

For this reason, we should stay clear from any relation to "Default". Both in code & in tests. 
(Ideally, this PR's diff should not contain "default" at all 😉  )
Using "default" would be misleading until we really do support DEFAULT -- which _may_ even be never.

Reply:
Wait. After second/third/fourth thought I'm not so sure about this.
We can say that
* from engine perspective, column has a default and connector can fill this in (directly, or leveraging external system to do that)
* when column has no explicit default, its default is NULL

@martint is this a correct nomenclature?

@Praveen2112 let me think more about this. Sorry for delay & appreciate your patience.